packages/fxa-auth-server/lib/oauth/db/index.js

```diff
@@ -80,11 +85,11 @@ class OauthDB {
     return db._getAccessToken(id);
   }
 
-  async removeAccessToken(id) {
-    const done = await this.redis.removeAccessToken(id);
+  async removeAccessToken(token) {
```

Comment:
`removeRefreshToken` needs to know both the `uid` and `tokenId`, and the `/destroy` endpoint generically calls either `removeAccessToken` or `removeRefreshToken` depending on the type of token. To keep that simple, I opted ti make both of them take the full token object rather than just the token id.